study-frontend/src/Components/Menu.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react'
+import { routes } from '../constants';
+import { useNavigate } from 'react-router-dom';
+import MenuButton from './MenuButton';
+
+type Props = {
+  handleFaBar: () => void
+}
+
+
+export const Menu = ({handleFaBar} : Props) => {
+  return (
+    <nav >
+      <div className='sideBar'>
+        {/* <div className='topSideBar'>
+            <h2 >Study</h2>
+        </div> */}
+        
+        <div className='downSideBar'>
```

Comment:
Do not write css classes like Js example css syntax
`li.spacious {
  margin: 2em;
}
li.spacious.elegant {
  margin: 2em;
}
.class_name { style properties }
[class~=class_name] { style properties }
.yellow-bg {
  background: #ffa;
}`

`<p class="red yellow-bg">
  This paragraph has red text and a yellow background.
</p>`

https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors